error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ReactorRules.java

```diff
@@ -144,6 +144,18 @@ Mono<S> after(Mono<T> mono, S object) {
     }
   }
 
+  static final class FluxTakeGenerationLimit<T> {
+    @BeforeTemplate
+    Flux<T> before(Flux<T> flux, long n) {
+      return flux.take(n);
+    }
+
+    @AfterTemplate
+    Flux<T> after(Flux<T> flux, long n) {
+      return flux.take(n, /* limitRequest= */ true);
```

Comment:
Gotta love the `self-check` profile, didn't expect this to show up in the after-template.

Fun fact; if we really want to add this comment in the actual output of applying a Refaster rule, we can use `Refaster#emitComment` or `Refaster#emitCommentBefore`.